Resources/Prototypes/Datasets/ion_storm.yml

```
@@ -5,986 +5,986 @@
+  - СЕМЬДЕСЯТ
+  - ВОСЕМЬ
+  - ВОСЕМЬДЕСЯТ
+  - ДЕВЯТЬ
+  - ДЕВЯНОСТО
 
 - type: dataset
   id: IonStormNumberMod
   values:
-  - BAZILLION
-  - BILLION
-  - BILLION FAFILLION GAJILLION SHAB-AB-DOOD-ILLION
-  - HUNDRED
-  - MILLION
-  - QUADRILLION
-  - THOUSAND
-  - TRILLION
+  - СОТЕН
+  - ТЫСЯЧ
+  - МИЛЛИОНОВ
+  - МИЛЛИАРДОВ
+  - ТРИЛЛИОНОВ
+  - КВАДРИЛЛИОНОВ
+  - СТО ТЫЩ МИЛЛИОНОВ
+  - БИЛИОН ФАФИЛИОНОВ ГАЗИЛИОНОВ ШАБ-АБ-ДУД-ЛОНОВ
 
 # Objects are anything that can be found on the station or elsewhere, plural.
 - type: dataset
   id: IonStormObjects
   values:
-  - AIRLOCKS
-  - ARCADE MACHINES
-  - AUTOLATHES
-  - BACKPACKS
-  - BANANA PEELS
-  - BEAKERS
-  - BEARDS
-  - BELTS
-  - BERETS
-  - BIBLES
-  - BODY ARMOR
-  - BOMBS
-  - BOOKS
-  - BOOTS
-  - BOTTLES
-  - BOXES
-  - BRAINS
-  - BRIEFCASES
-  - BUCKETS
-  - CABLE COILS
-  - CAMERAS
-  - CANDLES
-  - CANDY BARS
-  - CANISTERS
-  - CAT EARS
-  - CATS
-  - CELLS
-  - CHAIRS
-  - CHEMICAL DISPENSERS
-  - CHEMICALS
-  - CLONING EQUIPMENT
-  - CLONING PODS
-  - CLOSETS
-  - CLOTHES
-  - CLOWN CLOTHES
-  - COFFINS
-  - COLLECTABLES
-  - COMPUTERS
-  - CONTRABAND
-  - CORGIS
-  - CORPSES
-  - COSTUMES
-  - CRATES
-  - CRAYONS
-  - CROWBARS
-  - DEFIBRILLATORS
-  - DISABLERS
-  - DOORS
-  - DRONES
-  - EARS
-  - EMAGS
-  - ENGINES
-  - EQUIPMENT
-  - ERRORS
-  - EXOSKELETONS
-  - EXPERIMENTORS
-  - EXPLOSIVES
-  - EYEWEAR
-  - FEDORAS
-  - FIRE AXES
-  - FIRE EXTINGUISHERS
-  - FIRESUITS
-  - FLAMETHROWERS
-  - FLASHES
-  - FLASHLIGHTS
-  - FLOOR TILES
-  - FREEZERS
-  - GAS MASKS
-  - GLASS SHEETS
-  - GLOVES
-  - GUNS
-  - HAIRDOS
-  - HANDCUFFS
-  - HATS
-  - HEADS
-  - HEADSETS
-  - HELMETS
-  - HORNS
-  - ID CARDS
-  - INSULATED GLOVES
-  - JETPACKS
-  - JUMPSUITS
-  - LASERS
-  - LIGHT BULBS
-  - LIGHTS
-  - LOCKERS
-  - MACHINES
-  - MECHAS
-  - MEDICAL TOOLS
-  - MEDKITS
-  - MESONS
-  - MIME CLOTHES
-  - MINING TOOLS
-  - MULTITOOLS
-  - ORES
-  - OXYGEN TANKS
-  - PACKETS
-  - PAIS
-  - PANTS
-  - PAPERS
-  - PARTICLE ACCELERATORS
-  - PDAS
-  - PENS
-  - PETS
-  - PIPES
-  - PLANTS
-  - POSITRONIC BRAINS
-  - PUDDLES
-  - RACKS
-  - RADIOS
-  - RCDS
-  - REFRIGERATORS
-  - REINFORCED WALLS
-  - ROBOTS
-  - SCREWDRIVERS
-  - SEEDS
-  - SHOES
-  - SHUTTLES
-  - SINGULARITIES
-  - SINKS
-  - SKELETONS
-  - SOLAR PANELS
-  - SOLARS
-  - SPACE STATIONS
-  - SPACESUITS
-  - STEEL SHEETS
-  - STUN BATONS
-  - SUITS
-  - SUNGLASSES
-  - SUPPERMATTER SHARDS
-  - SWORDS
-  - SYRINGES
-  - TABLES
-  - TANKS
-  - TELECOMMUNICATION EQUIPMENTS
-  - TELEPORTERS
-  - TOILETS
-  - TOOLBELTS
-  - TOOLBOXES
-  - TOOLS
-  - TOYS
-  - TUBES
-  - VEHICLES
-  - VENDING MACHINES
-  - WELDERS
-  - WINDOWS
-  - WIRECUTTERS
-  - WIZARD ROBES
-  - WRENCHES
+  - ШЛЮЗЫ
+  - АРКАДНЫЕ АВТОМАТЫ
+  - АВТОЛАТЫ
+  - РЮКЗАКИ
+  - БАНАНОВЫЕ КОЖУРКИ
+  - МЕНЗУРКИ
+  - БОРОДЫ
+  - ПОЯСА
+  - БЕРЕТЫ
+  - БИБЛИИ
+  - БРОНЕЖИЛЕТЫ
+  - БОМБЫ
+  - КНИГИ
+  - БОТИНКИ
+  - БУТЫЛКИ
+  - КОРОБКИ
+  - МОЗГИ
+  - ПОРТФЕЛИ
```

```suggestion
  - ЧЕМОДАНЫ
```
у нас они так называются